astropy/nddata/ccddata.py

```diff
@@ -535,10 +535,11 @@ def fits_ccddata_reader(filename, hdu=0, unit=None, hdu_uncertainty='UNCERT',
     filename : str
         Name of fits file.
 
-    hdu : int, optional
-        FITS extension from which CCDData should be initialized. If zero and
-        and no data in the primary extension, it will search for the first
-        extension with data. The header will be added to the primary header.
+    hdu : int, str, tuple of (string, int), or BaseHDU, optional
```

Comment:
As mentioned previously, I don't think we can pass an HDU here.
```suggestion
    hdu : int, str, tuple of (str, int), optional
```

Reply:
Also the RTD build is stuck so I will commit this to this if a new build works.

Reply:
I thought I have committed your suggestion....

Reply:
I made a second one after checking what CCDdata does and should probable have removed the first one, sorry for the confusion.